rapidxml.hpp

```diff
@@ -689,7 +690,7 @@ namespace rapidxml
         //! <br><br>
         //! Use value_size() function to determine length of the value.
         //! \return Value of node, or empty string if node has no value.
-        Ch *value() const
+        const Ch *value() const
```

Comment:
We should at least have a non-cost value() overload that returns a non-const Ch*

(Even then I'm not all too happy about breaking the public interface, especially since iterators etc. only return const nodes/attributes.)

Reply:
It's not good idea to change the value and there is always static_cast or some dirty tricks to break that rule ;-)